src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,340 @@
+
+## Sampling Transactions
+
+When you enable sampling for APM events in Sentry, you choose a percentage of collected transactions to send to sentry. For example, if you had an endpoint that received 1000 requests per minute, a sampling rate of 0.25 would result in 250 transactions (25%) being sent to Sentry.
+
+Sampling enables you to collect traces on a subset of your traffic and extrapolate to the total volume. Furthermore, enabling sampling allows you to control the volume of data you send to Sentry and lets you better manage your costs. If you don't have a good understanding of what sampling rate to choose, we recommend you start low and gradually increase the sampling rate as you learn more about your traffic patterns and volume.
+
+When you have multiple projects collecting APM data, Sentry utilizes 'head-based' sampling to ensure that once a sampling decision has been made, that decision is propagated to each application or project involved in the transaction. If your applications have multiple entry points, you should aim to choose consistent sampling rates. Choosing different sampling rates can bias your results. Sentry does not support 'tail-based' sampling at this time.
+
+If you enable APM collection for a large portion of your traffic, you may exceed your organization's [Quotas and Rate Limits](https://docs.sentry.io/accounts/quotas/).
+
+## Setting Up Tracing
+
+[ALERT BOX: Supported SDKs:
+
+- JavaScript Browser
+- Node.js
+- Python version ≥ 0.11.2 ]
+
+Getting started with Sentry's APM features is a three step process. If you've already created a Sentry account and installed an SDK, skip to step 2a:
+
+1. [Sign up for an account](https://sentry.io/signup/)
+2. [Install your SDK](https://docs.sentry.io/error-reporting/quickstart/?platform=browser#pick-a-client-integration)
+
+    2a. Add APM integration
+
+        code snippet for customers to get started with APM
+
+    2b. Configure instrumentation, etc.
+
+3. [Configure](https://docs.sentry.io/error-reporting/quickstart/?platform=browser#configure-the-sdk) SDK
+
+### Python
+
+To send any traces, set the `traces_sample_rate`to a nonzero value. The following configuration will capture 10% of all your transactions:
+
+    import sentry_sdk
+        
+    sentry_sdk.init("___PUBLIC_DSN___", traces_sample_rate=0.1)
+
+**Automating Traces**
+
+Many integrations for popular frameworks automatically capture traces. If you already have any of the following frameworks set up for error reporting, you will start to see traces immediately:
+
+- All WSGI-based frameworks (Django, Flask, Pyramid, Falcon, Bottle)
+- Celery
+- Redis Queue (RQ)
+
+Spans are available for the following operations within a transaction:
+
+- Database that uses SQLAlchemy or the Django ORM
+- HTTP requests made with `requests` or the stdlib
+- Spawned subprocesses
```

Comment:
Can you add this (lmk if it needs clarification):

>If you want to automatically enable all relevant transactions, you can do:

```python
import sentry_sdk

sentry_sdk.init("___PUBLIC_DSN___", _experiments={"auto_enabling_integrations": True})
```

We want to mark this as alpha-in-alpha somehow... it's an alternative way of instrumentation that spawned as an experiment out of the APM alpha.